docs/stdlib/numbers.rst

```diff
@@ -446,11 +464,11 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: modulo mod division
 
-    Remainder from division (modulo).
+    Retrieves the remainder from a division operation. (modulo)
 
-    This is the remainder from floor division. Just as is
-    the case with :eql:op:`//<floordiv>` the result type of the
-    remainder operator corresponds to the operand type:
+    This is also the remainder from floor division. Just as is
+    the case with the :eql:op:`//<floordiv>` operator, the resulting type of
```

Comment:
```suggestion
    the case with the :eql:op:`//<floordiv>` operator, the result type of
```

- "resulting type" implies the result _is_ a type. The type of the result or "result type" is what we are talking about here.